pepper-apis/pom.xml

```diff
@@ -17,8 +17,10 @@
         <jdbi.version>3.28.0</jdbi.version>
         <itext.version>7.1.4</itext.version>
         <itext.licensekey.version>3.0.3</itext.licensekey.version>
-        <liquibase.version>3.6.3</liquibase.version>
+        <liquibase.version>4.8.0</liquibase.version>
```

Comment:
Added versions of various dependencies here so they will be easier to find going forward.